src/Database/Dongle.php

```diff
@@ -65,10 +65,6 @@ public function parse($sql)
     public function parseGroupConcat($sql)
     {
         $result = preg_replace_callback('/group_concat\((.+)\)/i', function ($matches) {
-            if (!isset($matches[1])) {
-                return $matches[0];
-            }
```

Comment:
Why was this required?

Reply:
It's pointless. The first match will always be present if the callback is run, because the callback won't be run if there's no expression match.